app/src/main/java/com/apator/map/helpers/formater/DoubleToString.kt

```diff
@@ -0,0 +1,11 @@
+package com.apator.map.helpers.formater
+
+object DoubleToString {
```

Comment:
Poprawić nazwę

Reply:
W sumie to działa jak serializer więc powinien mieć 2 metody do serializacji i deserializacji